docs/source/tools/sandbox.rst

```diff
@@ -226,3 +226,391 @@ Command-line reference
+
+- ``--metrics-reporting-interval``: metrics are pre-aggregated on the sandbox and sent to
+  the reporter, this option allows the user to set the interval. The formats accepted are based
+  on the ISO-8601 duration format ``PnDTnHnMn.nS`` with days considered to be exactly 24 hours.
+  The default interval is 10 seconds.
+
+Types of metrics
+================
+
+This is a list of type of metrics with all data points recorded for each.
+Use this as a reference when reading the list of metrics.
+
+Gauge
+-----
+
+An individual instantaneous measurement.
+
+Counter
+-------
+
+Number of occurrences of some event.
+
+Meter
+-----
+
+A meter tracks the number of times a given event occurred. The following data
+points are kept and reported by any meter.
+
+- ``<metric.qualified.name>.count``: number of registered data points overall
+- ``<metric.qualified.name>.m1_rate``: number of registered data points per minute
+- ``<metric.qualified.name>.m5_rate``: number of registered data points every 5 minutes
+- ``<metric.qualified.name>.m15_rate``: number of registered data points every 15 minutes
+- ``<metric.qualified.name>.mean_rate``: mean number of registered data points
+
+Histogram
+---------
+
+An histogram records aggregated statistics about collections of events.
+The exact meaning of the number depends on the metric (e.g. timers
+are histograms about the time necessary to complete an operation).
+
+- ``<metric.qualified.name>.mean``: arithmetic mean
+- ``<metric.qualified.name>.stddev``: standard deviation
+- ``<metric.qualified.name>.p50``: median
+- ``<metric.qualified.name>.p75``: 75th percentile
+- ``<metric.qualified.name>.p95``: 95th percentile
+- ``<metric.qualified.name>.p98``: 98th percentile
+- ``<metric.qualified.name>.p99``: 99th percentile
+- ``<metric.qualified.name>.p999``: 99.9th percentile
+- ``<metric.qualified.name>.min``: lowest registered value overall
+- ``<metric.qualified.name>.max``: highest registered value overall
+
+Histograms only keep a small *reservoir* of statistically relevant data points
+to ensure that metrics collection can be reasonably accurate without being
+too taxing resource-wise.
+
+Unless mentioned otherwise all histograms (including timers, mentioned below)
+use exponentially decaying reservoirs (i.e. the data is roughly relevant for
+the last five minutes of recording) to ensure that recent and possibly
+operationally relevant changes are visible through the metrics reporter.
+
+Note that ``min`` and ``max`` values are not affected by the reservoir sampling policy.
+
+You can read more about reservoir sampling and possible associated policies
+in the `Dropwizard Metrics library documentation <https://metrics.dropwizard.io/4.1.2/manual/core.html#man-core-histograms/>`__.
+
+Timers
+------
+
+A timer records all metrics registered by a meter and by an histogram, where
+the histogram records the time necessary to execute a given operation (measured
+in milliseconds, unless otherwise specified).
```

Comment:
The measurement itself is taken in nanoseconds, but the graphite reporter reports durations in milliseconds. This could be changed in `MetricsReporter.scala` by using `GraphiteReporter.Builder#convertDurationsTo()`.

Reply:
I was referring to the unit, not the precision. How could I phrase it better?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6662/commits/995f5b536bfc6336a5e05d08c10ebb8ee0ea88e0